Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -61,23 +93,49 @@ struct Footer: View {
 let footerReducer: Reducer<
   FooterState,
   FooterAction,
-  Void
+  SidebarEnvironment
 > = Reducer.combine([
   footerActionMenuReducer.pullback(
-    state: \.actionButton,
+    state: \FooterState.actionButton,
     action: CasePath(FooterAction.actionButton),
-    environment: { $0 }
+    environment: { _ in () }
   ),
   footerAvatarReducer.pullback(
-    state: \.avatar,
+    state: \FooterState.avatar,
     action: CasePath(FooterAction.avatar),
+    environment: { _ in () }
+  ),
+  editProfileReducer.pullback(
```

Comment:
Would you mind referencing the ticket `https://github.com/pointfreeco/swift-case-paths/issues/86` here. Hopefully we can remove that workaround in the future know why it exists in the meanwhile.

Reply:
I guess I won't need to 🙃

Reply:
I'll update the dependency instead